Example/Standard Integration/CheckoutViewController.swift

```diff
@@ -250,32 +250,42 @@ See https://stripe.com/docs/testing.
                                                                     completion(error ?? NSError(domain: StripeDomain, code: 123, userInfo: [NSLocalizedDescriptionKey: "Unable to parse clientSecret from response"]))
                                                                     return
                                                                 }
-                                                                // Retrieve the PaymentIntent
-                                                                STPAPIClient.shared().retrievePaymentIntent(withClientSecret: clientSecret, completion: { (paymentIntent, retrieveError) in
-                                                                    guard let paymentIntent = paymentIntent else {
-                                                                        completion(retrieveError ?? NSError(domain: StripeDomain, code: 123, userInfo: [NSLocalizedDescriptionKey: "Unable to parse payment intent from response"]))
-                                                                        return
-                                                                    }
-
-                                                                    if paymentIntent.status == .requiresAction {
-                                                                        STPPaymentHandler.shared().handleNextAction(forPayment: paymentIntent
-                                                                            , with: self, completion: { (status, handledPaymentIntent, actionError) in
-                                                                                switch (status) {
-
-                                                                                case .succeeded, .failed:
-                                                                                    completion(actionError)
-
-                                                                                case .canceled:
-                                                                                    completion(NSError(domain: StripeDomain, code: 123, userInfo: [NSLocalizedDescriptionKey: "User canceled authentication."]))
+                                                                STPPaymentHandler.shared().handleNextAction(forPayment: clientSecret, with: self) { (status, handledPaymentIntent, actionError) in
+                                                                    switch (status) {
+                                                                    case .succeeded:
+                                                                        if (handledPaymentIntent!.status == .requiresConfirmation) {
```

Comment:
we probably don't want to encourage force unwrapping here, but won't block on it

Reply:
I guess I'll add a `guard`.  It's unfortunate the obj-c based API allows all these invalid states like status == .succeeded && handledPaymentIntent == nil.